tests/unit/dojson/test_dojson_common_base.py

```diff
@@ -382,3 +382,39 @@ def test_urls_from_marcxml_multiple_8564():
     result = clean_record(hep.do(create_record(snippet)))
 
     assert expected == result['urls']
+
+
+def test_for_980_case():
+    """Test for 980 case.
+
+    """
+    snippet = (
+        '<record>'
+        '  <datafield tag="980" ind1=" " ind2=" ">'
+        '    <subfield code="a">Test_a1</subfield>'
+        '    <subfield code="a">Test_a2</subfield>'
+        '    <subfield code="b">Something</subfield>'
+        '    <subfield code="c">DELETED</subfield>'
+        '  </datafield>'
+        '</record>'
+    )
+
+    expected_json = [
+        {
+            'deleted': True,
+            'primary': 'Test_a1',
+            'secondary': 'Something',
+        },
+    ]
+    expected_marc = [
+        {
+            'c': True,
```

Comment:
Be careful: here it should be `'c': 'DELETED'` so something is not correct right now.